scripts/todo.js

```diff
@@ -40,17 +40,17 @@ window.onload = function() {
   note.appendChild(createListItem('App initialised.'));
   // In the following line, you should include the prefixes of implementations you want to test.
   window.indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
-  // DON'T use "var indexedDB = ..." if you're not in a function.
+  // DON'T use 'var indexedDB = ...' if you're not in a function.
   // Moreover, you may need references to some window.IDB* objects:
   window.IDBTransaction = window.IDBTransaction || window.webkitIDBTransaction || window.msIDBTransaction;
   window.IDBKeyRange = window.IDBKeyRange || window.webkitIDBKeyRange || window.msIDBKeyRange;
   // (Mozilla has never prefixed these objects, so we don't need window.mozIDB*)
 
   // Let us open our database
-  const DBOpenRequest = window.indexedDB.open("toDoList", 4);
+  const DBOpenRequest = window.indexedDB.open('toDoList', 4);
 
   // Gecko-only IndexedDB temp storage option:
-  // var request = window.indexedDB.open("toDoList", {version: 4, storage: "temporary"});
+  // var request = window.indexedDB.open('toDoList', {version: 4, storage: 'temporary'});
 
   // these two event handlers act on the database being opened successfully, or not
   DBOpenRequest.onerror = function(event) {
```

Comment:
Let's use arrow functions for anonymous functions where possible.
```suggestion
  DBOpenRequest.onerror = (event) => {
```

Reply:
Please do the same on line 60 (GitHub UI doesn't allow me to make a suggestion there 😢 )